api.md

```diff
@@ -919,6 +922,48 @@ The expected output:
 
 ---
 
+## interpolate() [](interpolate)
+
+The `interpolate` does linear interpolation for missing values.
```

Comment:
"The 'interpolate' _function_ ..."?  Also, maybe the intro sentences can be combined as: "The interpolate function does linear interpolation for missing values.  It can only be used ..."

Reply:
fixed